src/backends/sentry_backend_crashpad.cpp

```diff
@@ -426,6 +427,22 @@ sentry__crashpad_backend_last_crash(sentry_backend_t *backend)
     return crash_time;
 }
 
+static void
+sentry__crashpad_backend_prune_database(sentry_backend_t *backend)
+{
+    crashpad_state_t *data = (crashpad_state_t *)backend->data;
+
+    // We want to eagerly clean up reports older than 2 days, and limit the
+    // complete database to a maximum of 8M. That might still be a lot for
+    // an embedded use-case, but minidumps on desktop can sometimes be quite
+    // large.
+    data->db->CleanDatabase(60 * 60 * 24 * 2);
```

Comment:
2 days still seems a long time to me, why not immediate?  Or are we at risk of removing unsent minidumps?

Reply:
crashpad has 3 states essentially: `new`, `pending` and `completed`. The function cleans pending and completed minidumps. I would have to look into what it does exactly on upload (create as `new`, move into `pending` when it starts uploading?)
Keeping the "most recent" minidump might be fair to do. At least the database should not grow seemingly unbounded.

Reply:
in theory crashpad tries to upload as soon as the crash happens right?  so this will only find new or pending minidumps if that failed somehow.  I guess 2 days is reasonable, it's just really though to come up with something that suits all :)

Reply:
I'd strongly prefer to wipe the minidumps as soon as they get uploaded. 
What about making the parameters configurable though API calls?